docs/pages/manage/actionsettings.md

```diff
@@ -0,0 +1,19 @@
+{% include navigation.html %}
+
+# Action Configuration
+
+Certain actions allow for a global configuration. These configurations are defined under the Action level.
+
```

Comment:
This is quite high level -- Perhaps we could add more information as introduction? "Scripts can be easily build by defining and sequentially executing actions. These actions are defined through action types which allow for a global configuration. These configurations/types are defined...." -- just thinking a bit here

Reply:
Indeed, we can elaborate on this more in a next phase :)